Model/User.hs

```diff
@@ -76,14 +75,11 @@ credsToUser eplanId Creds{..} = User
     eprofile = extraToProfile credsPlugin credsExtra
 
 extraToProfile :: Text -> [(Text, Text)] -> Either Text Profile
-extraToProfile "dummy" _ = Right dummyProfile
+extraToProfile "dummy" extra = githubProfile extra
```

Comment:
Does this work because GitHub/dummy logins happen to have the same `extra` mappings?